taiga_contrib_ldap_auth/services.py

```diff
@@ -62,17 +67,29 @@ def ldap_login_func(request):
     # (or any other attribute)
     login_input = request.DATA.get('username', None)
     pass_input = request.DATA.get('password', None)
-
-    # TODO: sanitize before passing to LDAP server?
-    username, email, full_name = connector.login(login = login_input,
-                                                 password = pass_input)
-
-    user = register_or_update(username = username,
-                              email = email,
-                              full_name = full_name)
-
-    data = make_auth_response_data(user)
-    return data
+    try:
+        # TODO: make sure these fields are sanitized before passing to LDAP server!
+        username, email, full_name = connector.login(login = login_input, password = password_input)
+    except connector.LDAPUserLoginError as ldap_error:
```

Comment:
Perhaps should be `LDAPConnectionError` instead?.. Wouldn't this otherwise allow logging in with the old local password if password in LDAP has changed?